src/__screenshot_tests__/skeletons-screenshot-test.tsx

```diff
@@ -0,0 +1,89 @@
+import {openStoryPage, screen} from '../test-utils';
```

Comment:
There is a lot of garbage in the `__image_snapshots__` directory. Probably caused by updating the screenshots-tests with titles, etc.

I took your branch and there should be just 17 screenshots:
![image](https://user-images.githubusercontent.com/579417/190975362-8d663fe8-03f3-45a2-b755-9603d739a07c.png)